CMakeLists.txt

```diff
@@ -40,9 +40,13 @@ if("${os_name}" STREQUAL "clear-linux-os")
     set(CMAKE_INSTALL_SYSCONFDIR "usr/share/defaults/etc")
 endif()
 
+option (BUILD_CMRTLIB "Build and Install cmrtlib together with media driver" ON)
```

Comment:
@chivakker: this commit looks ok to me. How about you?

Reply:
Can default be OFF?  Users should intentionally know they are enabling it.  Otherwise it looks okay.

Reply:
cmrtlib and media_driver are both in this repo, so we still prefer this be ON by default.